packages/module/src/LogSnippet/LogSnippet.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { CodeBlock, CodeBlockAction, CodeBlockCode, ClipboardCopyButton, Button, ClipboardCopy } from '@patternfly/react-core';
+
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  logSnippet: {
+    borderLeft: '2px solid var(--pf-v5-global--danger-color--100)',
+    display: 'flex',
```

Comment:
There is a PF layout we should be able to use rather than hardcoding layouts into the CSS. https://www.patternfly.org/layouts/flex